src/components/data/DataTable.js

```diff
@@ -237,7 +238,8 @@ const DataTable = props => {
 
 
   // Render
-  const columnSettings = applyColumnSettings(columnState || [], entityMetadata[entityType].attributeNames)
+  // If there is an active cross table search, temporarily reset the column settings to show all columns
+  const columnSettings = activeCrossTableTextFilter ? applyColumnSettings([], entityMetadata[entityType].attributeNames) : applyColumnSettings(columnState || [], entityMetadata[entityType].attributeNames)
```

Comment:
Since this overrides a user's table column settings, should we disable the settings button while a cross table search is active?

Reply:
Addressed here https://github.com/DataBiosphere/terra-ui/pull/3040/commits/33e50019c0700abc1464e3c77ac613bec4323b68